x-pack/plugins/cases/docs/openapi/components/schemas/connector_properties_servicenow_sir.yaml

```diff
@@ -0,0 +1,60 @@
+title: Create case request properties for a ServiceNow SecOps connector
+required:
+  - fields
+  - id
+  - name
+  - type
+description: Defines properties for connectors when type is `.servicenow-sir`.
+type: object
+properties:
+  fields:
+    description: An object containing the connector fields. If you want to omit any individual field, specify null as its value.
+    type: object
+    required:
+      - category
+      - destIp
+      - malwareHash
+      - malwareUrl
+      - priority
+      - sourceIp
+      - subcategory
+    properties:
+      category:
+        description: The category of the incident. 
+        type: string
+        nullable: true
+      destIp:
```

Comment:
The fields schema for SIR is here: https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/common/api/connectors/servicenow_sir.ts

```
destIp: boolean | null
malwareHash: boolean | null
malwareUrl: boolean | null
sourceIp: boolean | null
```

I think these are essentially indicating whether cases will send those particular fields rather than the values themselves.

Reply:
Good catch!  I've updated those data types in e07444fccc797dd4ea443e5113f366964a7d3078